contracts/token/ERC721/IERC721Enumerable.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.24;
+
+import "./IERC721Basic.sol";
+
+
+/**
+ * @title ERC-721 Non-Fungible Token Standard, optional enumeration extension
+ * @dev See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
+ */
+contract IERC721Enumerable is IERC721Basic {
```

Comment:
Is it possible to make this an `interface`?

Reply:
Interfaces cannot inherit, sadly.